src/vaadin-text-field-mixin.html

```diff
@@ -499,10 +499,10 @@
       const input = this.inputElement;
       const attributeNames = HOST_PROPS[type];
 
-      if (type === 'accessible') {
+      if (type === PROP_TYPE.ACCESSIBLE) {
         attributeNames.forEach((attr, index) => {
           this._setOrToggleAttribute(attr, attributesValues[index], input);
-          this._setOrToggleAttribute(`aria-${attr}`, attributesValues[index], input);
+          this._setOrToggleAttribute(`aria-${attr}`, attributesValues[index] ? 'true' : false, input);
```

Comment:
This makes it so that when setting `aria-*` attribute here (now only `invalid` -> `aria-invalid`) it is always cast to boolean and it either sets `aria-*="true"` or unsets the attribute when evaluating to false.